Modules/vc-module-catalog/VirtoCommerce.CatalogModule.Data/ExportImport2/ExportableCatalog.cs

```diff
@@ -0,0 +1,27 @@
+using System.Linq;
```

Comment:
Need to remove?

Reply:
No :-)

Reply:
Where this class is used? I cannot find the place, that's why i think we could remove it.

Reply:
FRomModel:
Languages?.Select